types/index.d.ts

```diff
@@ -187,6 +187,13 @@ declare namespace build {
      * @default 'default'
      */
     largeArrayMechanism?: 'default' | 'json-stringify'
+
+    /**
+     * Specify if null key-value are removed from serialised JSON output, for a smaller payload
+     *
+     * @default 'false'
+     */
+    stripNull?: boolean
```

Comment:
Will it be better to do in `ignoreValue: ('undefined' | 'null')[]`?
Then provide a function to build the ignore statement.

```js

function buildIgnoreValue(ignores) {
  let str = ''
  for(let i = 0; i < ignores.length; i++) {
    str += `value !== ${ignore}`
    if (i < ignores.length - 1) str += ' || '
  }
  return str
}